do.sh

```diff
@@ -163,13 +198,52 @@ function install_ovn_fake_multinode() {
 
     # Clone repo locally
     clone_component ovn-fake-multinode ${ovn_fmn_repo} ${ovn_fmn_branch} || rebuild_needed=1
-    clone_component ovs ${ovs_repo} ${ovs_branch} || rebuild_needed=1
-    clone_component ovn ${ovn_repo} ${ovn_branch} || rebuild_needed=1
+
+    if [ -n "$RPM_OVS" ]
```

Comment:
Do you mind adding a section in the readme about these new variables that allow deployments with prebuilt RPMs? I.e., documenting `RPM_OVS`, `RPM_OVN_COMMON`, `RPM_OVN_CENTRAL`, `RPM_OVN_HOST`, `RPM_SELINUX`.

Reply:
sure, I will